support/src/main/java/com/yubico/yubikit/support/YubiKeyUsbProductId.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2022 Yubico.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.yubico.yubikit.support;
+
+import com.yubico.yubikit.core.Logger;
+import com.yubico.yubikit.management.UsbInterface;
+
+public enum YubiKeyUsbProductId {
```

Comment:
I think this name may be a bit long. Maybe `YubiKeyPid`, `UsbPid`, or something?